### DISCUSSION TITLE
Saving filter set in Collection

### DISCUSSION THREAD

**Discussion by artisak2** [deckerst/aves#1106] [Q&A] [answered]
Is there a way to save a filtered collection you are currently viewing with all filters applied?

I know that i can "add a shortcut" to the launcher of the collection I am viewing with the filters applied, but I do not see away to save and access this in the app.

For example:
DirA
|
|__DirB
|
|__DirC
|
|__DirD

I would like to find a way to quickly view everthing in DirA but filter out DirC and anything with TAG: "xyz" etc.. etc..

I ask because there are some cases where i find myself repeating, and would find this useful.

Thanks!

edit: Formatting

In fact, if there is a way to have a page with all saved filter sets as "Home" when app opens, that would be even better

**Comment by deckerst** [comment, marked as answer]
At the moment there's no way to save a filter set for later reuse, besides using it as home, or adding it as a shortcut.
I thought there was an issue requesting this feature but I can't find it.

Anyway, I agree it would be useful. In some other apps, they turn that kind of filter set into a "smart album" or "virtual album", but I'd probably treat it as another filter itself, so it can be used like the existing standard ones.

**Reply by artisak2** [reply]
Yes, ive used those "smart albums" before and i found them to be very useful.
Thanks for the answer!
